src/status_im/init/core.cljs

```diff
@@ -203,8 +203,18 @@
 
 (defn login-only-events [cofx address stored-pns]
   (fx/merge cofx
-            {:notifications/request-notifications-permissions nil}
-            (navigation/navigate-to-cofx :home nil)
+            (cond->
+             {:notifications/request-notifications-permissions nil}
+
+              platform/ios?
+              ;; on ios navigation state might be not initialized yet when
+              ;; navigate-to call happens.
+              ;; That's why it should be delayed a bit.
+              (assoc :dispatch-later
+                     [{:ms       1
```

Comment:
is this 1 ms arbitrary or you just want it to be executed in the next tick?

Reply:
Right, we need to execute it on the next tick.

Reply:
is there a callback when navigation state is initialized? In any case it seems like this should be handled in navigation namespace

Reply:
there is no callback

There is probably a better way to handle this but this should be enough for fixing a bug before release. We can invest more time in "proper" fix later.

Reply:
can you please add a TODO in the code and create a ticket for it then? Because I really wouldn't want this to become permanent code

Reply:
sure

Reply:
done